ts/util/text-input.ts

```diff
@@ -58,9 +72,31 @@ export class TextInput {
     }
   }
 
-  public getTag() {
+  private async loadAutocomplete(queryUrl: string) {
+    const val = this.textField.value;
+    if (val.length < 1) {
+      // results are only displayed for input > **2** characters
+      return;
```

Comment:
condition and comment don't seem to match?

Reply:
doc here is correct due to the nature of how keypress events work (i moved autocomplete querys to the keypress event listener so it doesnt autocomplete again once you select from the datalist)

Reply:
Oh, I see. Can improve this by using the `input` event instead of `keydown` and moving the `loadAutocomplete` call there. Then you'll get the actual full value here, so you can check `val.length < 2`

Should also clear the datalist before returning here. Otherwise, if you type "WG", clear the input and then enter "T" it still shows the entries for "WG"

Reply:
moving `loadAutocomplete` to a `input` event reintroduces the duped autocomplete behaviour though. I had that originally before moving it to the `keydown` event